Lisp Tree View doc

madkins23 opened the pull request:
This is a first draft in case you really don't want me to do this.

I didn't really get the Lisp Tree View when I (admittedly) skimmed the Readme for Alive. As I started going through the code I realized how useful it is. So I wanted to add some documentation focused on the Lisp Tree View and the views it contains.

If this makes sense to you I will probably clean it up some and try to find a way to add images of the icons to the text. If you're just not interested just reject the PR.

madkins23 commented on README.md:
```diff
@@ -80,25 +80,98 @@ Syntax highlighting is done using semantic tokens. This is mainly to avoid regex
+
+For each package in the list there are actions.
+Clicking on the minus icon at the right removes the package.
+Clicking on the package itself expands to a list of symbols in that package.
```
Since the Packages view drills down into package symbols, have you considered adding an inspect button to each symbol item? That might be easier than the way the Inspector view works.

nobody-famous replied:
Yeah, I've had some grandiose ideas for the tree view. :-) The main idea is that it should be an active view into the running image. Having an inspector button would be good. I've also thought of having navigation like the file tree. You click on a symbol and it jumps to where it's defined. Stuff like that.

The current inspector thing is so you can enter any lisp expression and it'll open an inspector with the result. There's an inspect link at the bottom of the hover for symbols, but I'm not sure anybody ever sees it.

madkins23 commented:
I went ahead and added some images to the README. I've been trying to avoid rewriting your doc too much but I've moved some of it around so that it made more sense to me. _Mea culpa_.
